src/openms/include/OpenMS/ANALYSIS/QUANTITATION/ProteinResolver.h

```diff
@@ -164,50 +164,50 @@ namespace OpenMS
     */
     void resolveID(std::vector<PeptideIdentification> & peptide_identifications);
 
-    /**
-      @brief NOT IMPLEMENTED YET
-
-      @param protein_nodes
-      @param peptide_nodes
-      @param reindexed_proteins
-      @param reindexed_peptides
-      @param peptide_identifications
-      @param output
-    */
+    // /**
```

Comment:
is this correct?

Reply:
yes. If you leave the doxygen comments like they were, all these parameters get attributed to the first active function below...